akka-stream/src/main/scala/akka/stream/javadsl/Sink.scala

```diff
@@ -116,6 +117,12 @@ object Sink {
     new Sink(scaladsl.Sink.actorSubscriber(props))
 
   /**
+   * Creates a `Sink` which uses Akka-IO's unconnected UDP mode to emit each ByteString it receives to the given target address.
+   */
+  def simpleUdp(host: String, port: Int): Sink[ByteString, Unit] =
```

Comment:
Nice idea for signature of the unconnected one.
I wanted this one to become the connected one anyway :-)